### ISSUE TITLE
Pokemon Sword crash/frozen randomly

### ISSUE THREAD

**Issue by ss8688** [yuzu-emu/yuzu#7722]
Yuzu crash or forzen randomly when runing Pokemon Sword.
It might happens every 30 minutes on average，even freeze the system
[yuzu_log.txt.old.txt](https://github.com/yuzu-emu/yuzu/files/7876858/yuzu_log.txt.old.txt)

I am not sure if it is a right log file. It might be overwritten. I will send a new log file after it crashes again.

**Comment by ss8688** [comment]
Frozen again, The graphic Frozen, but the sound is still worlking.
There is a very high RAM usage 
<img width="1122" alt="屏幕截图 2022-01-16 210349" src="https://user-images.githubusercontent.com/59367202/149661267-755a05de-1531-46d8-8a99-9fc8683c2fe6.png">

**Comment by ss8688** [comment]
The log file is too big
<img width="328" alt="image" src="https://user-images.githubusercontent.com/59367202/149661647-0b424118-99cb-4544-881d-827495dd0886.png">

**Comment by ss8688** [comment]
The log file is too big
<img width="328" alt="image" src="https://user-images.githubusercontent.com/59367202/149661647-0b424118-99cb-4544-881d-827495dd0886.png">

**Comment by ss8688** [comment]
<img width="1077" alt="image" src="https://user-images.githubusercontent.com/59367202/149661739-6fff45de-0a32-410a-a379-42f7b6ef6475.png">

**Comment by ss8688** [comment]
Get log here 
https://drive.google.com/file/d/15QccXk3NY7Ii4ZGAc3ewadKVEy9BLtIA/view?usp=sharing

**Comment by ss8688** [comment]
Yuzu has used 11GB RAM look like a memory leak?

**Comment by TheSnorlaxBoi** [comment]
I believe that was already established that SW/SH has memory leaks problems on yuzu.

**Comment by TheSnorlaxBoi** [comment]
Also, I have similar problem but for some reason I only get frezze in the Wild Areas, in the parts when there's a thunderstorm present. Don't know what's been causing the problem to be honest.

**Comment by G-Spawn** [comment]
Yes, this issue is expected. Please wait for further emulator improvement. Save often for now.

**Comment by ss8688** [comment]
Crash again， this time is just like the first log，different from the second one. It seems not leaks problems. Yuzu only used 1.5GB memory
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/7878976/yuzu_log.txt)
.

**Comment by ss8688** [comment]
This time it just crash, not frezze

**Comment by ss8688** [comment]
There seems 2 different problems?

**Comment by ss8688** [comment]
878 and Early Access 2402 both crashed

**Comment by ss8688** [comment]
It crashes again
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/7879199/yuzu_log.txt)

**Comment by ss8688** [comment]
Frezze again， memory of Yuzu goes up form 1.7G to 4G second after it Frezze 
<img width="1103" alt="image" src="https://user-images.githubusercontent.com/59367202/149728401-bb1e7d59-967f-4ad4-98ba-3f9f76f31faf.png">
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/7879945/yuzu_log.txt)
Yuzu seems frezze when using opengl，crash when using vulkan

**Comment by JustRed23** [comment]
#7737

**Comment by ARKseal** [comment]
I also see the memory leak, once the game freezes the memory usage increases at almost 500mb/s creating a 28+GB virtual memory file. My settings were OpenGL, GPU HIGH, Dock selected, BILINEAR, FXAA on Pokemon Sword.
The log file was filled with this

> [4491.532282] Core.ARM <Error> core\arm\dynarmic\arm_dynarmic_64.cpp:InterpreterFallback:86: Unimplemented instruction @ 0x2105E293A8 for 4175 instructions (instr = 00000000)

**Comment by Immersion95** [comment]
I have the same issue, what's the best practices to prevent it while waiting for a fix ?

**Comment by LOuroboros** [comment]
> Frozen again, The graphic Frozen, but the sound is still worlking.

I experienced this same issue about 3 or 4 times in different sessions myself too.

It doesn't look like it's hardware specific, seeing how people with far better hardware than me are also coming across it, so I'd like to leave here my log while understanding that my PC is below the minimum hardware requirements in case it can be helpful: 
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/9589338/yuzu_log.txt)

This log comes from booting up the game, trying to use the fishing rod in a water spot in Hulbury a couple of minutes after, and the game freezing in a similar fashion; an animation plays indefinitely while the music can still be listened to in the background.

Normally, the freezes happen at random points during a session *(normally after the 20 minutes mark)* and until now they've all been exactly the same *(keep trying to catch Pokémon and sooner or later one of those Poké Balls you throw will end up freezing the game after a shake or two)*, but never as early, so this is one weird exception.

https://user-images.githubusercontent.com/4485172/190814547-d9c7ee18-d739-4937-b620-bb82d9efe9ab.mp4

Here's my savefile too, in case it can help with the testing process.

[0100ABF008968000.zip](https://github.com/yuzu-emu/yuzu/files/9589561/0100ABF008968000.zip)

**Comment by artifall** [comment]
Any updates??? I have the same  problem

**Comment by nikhil-prabhu** [comment]
Can confirm that I'm facing a similar issue. After around 20-30 minutes of playing, Yuzu freezes and crashes.

### Analysis

#### Log File

[yuzu_log.txt.old.txt](https://github.com/yuzu-emu/yuzu/files/13725436/yuzu_log.txt.old.txt)

I couldn't really find any indications of errors in the log. I did notice however, that the log file seemingly cuts off at the end (Yuzu probably crashed halfway through writing to the logs?).

#### Observations

I noticed that Yuzu's memory usage kept fluctuating constantly between ~1GB to ~4GB (probably GC kicking in?). At no point did it exceed ~4GB in the first 20-30 minutes of playing.

[yuzu_memusage.webm](https://github.com/yuzu-emu/yuzu/assets/39860598/1f9e1616-bfbf-4a0c-b5ac-8305ff2f5d20)

After the 20-30 minute mark, Yuzu froze, and I noticed that the memory usage had gone up to 5.5GB, and then up to 6.3GB immediately after. It did not go past this, and Yuzu crashed after a few seconds.

![Screenshot_20231220_144507](https://github.com/yuzu-emu/yuzu/assets/39860598/59ce203f-e02a-499a-89ed-cf58f5de711c)
![Screenshot_20231220_144534](https://github.com/yuzu-emu/yuzu/assets/39860598/433c25d9-6d7d-4a1f-a2f1-d3226ed4acb0)

### Information

#### System Information

[sysinfo.txt](https://github.com/yuzu-emu/yuzu/files/13725538/sysinfo.txt) (Contains the output of `inxi -F`)

Operating System: openSUSE Tumbleweed 20231215
KDE Plasma Version: 5.27.10
KDE Frameworks Version: 5.112.0
Qt Version: 5.15.11
Kernel Version: 6.6.6-1-default (64-bit)
Graphics Platform: Wayland
Processors: 8 × AMD Ryzen 5 3400G with Radeon Vega Graphics
Memory: 29.3 GiB of RAM
Graphics Processor: AMD Radeon RX 6700 XT

#### Yuzu Information

Version: 1653 (2023-12-16) (Flatpak).
Settings: Default settings that came out of the box (only remapped controller inputs, which should be irrelevant).
Graphics API: Vulkan
Enable Gamemode (Linux): Yes
